packages/manager/modules/vrack/src/vrack-services/ip/delete/index.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+
+import template from './template.html';
+
+const moduleName = 'ovhManagerVrackAssignedIpServiceDeleteSubnetModal';
+
+angular
+  .module(moduleName, ['oui', 'pascalprecht.translate'])
+  .component('deleteIpv6SubnetModal', {
+    template,
+    controller: [
+      '$scope',
+      function() {
+        this.cancel = () => {
+          this.isOpenModal = false;
+        };
+      },
+    ],
```

Comment:
`$scope` injection is useless here.
Moreover, it's weird to have the close function defined here but not the open one.
As you're using an `onConfirm` function, sounds more logic to have also `onCancel` function